apps/dashboard/src/components/workflow-list.tsx

```diff
@@ -154,86 +153,71 @@ export const WorkflowList = () => {
           ) : (
             <>
               {workflowsQuery.data.workflows.map((workflow) => (
-                <>
-                  <TableRow key={workflow._id} className="relative">
-                    <TableCell className="font-medium">
-                      <div className="flex items-center gap-1">
-                        {workflow.origin === WorkflowOriginEnum.EXTERNAL && (
-                          <Badge className="rounded-full px-1.5" variant="warning-light">
-                            <BadgeContent variant="warning">
-                              <FaCode className="size-3" />
-                            </BadgeContent>
-                          </Badge>
-                        )}
-                        <TruncatedText
-                          className="cursor-pointer"
-                          text={workflow.name}
-                          onClick={() => {
-                            navigate(
-                              buildRoute(ROUTES.EDIT_WORKFLOW, {
-                                environmentId: currentEnvironment?._id ?? '',
-                                workflowId: workflow._id,
-                              })
-                            );
-                          }}
```

Comment:
i think this is removed by mistake, without this we wont be able to navigate to the edit workflow screen

Reply:
Maybe I did this on the rebase by accident. I'll fix it.